python/taichi/lang/ops.py

```diff
@@ -382,13 +584,39 @@ def logical_not(a):
 
 
 def random(dtype=float):
-    """The random function.
+    """Return a single random float/integer according to the specified data type.
+    Must be called in taichi scope.
+
+    For float types return a random float in the half-open interval [0, 1).
+
+    For int types return a random integer in the half-open interval [0, 2^32) for 32-bit,
+    or a random integer in the half-open interval [0, 2^64) for 64-bit.
 
     Args:
-        dtype (DataType): Type of the random variable.
+        dtype (:mod:`~taichi.types`): Type of the required random value.
 
     Returns:
-        A random variable whose type is `dtype`.
+        A random value with type `dtype`.
+
+    Example::
+
+        >>> @ti.kernel
+        >>> def main():
+        >>>     x = ti.random(float)
+        >>>      print(x)
+        0.090257
```

Comment:
this doesn't look right: you only defined a kernel here but didn't run it? Or maybe you wanted to put them in the comment?

Reply:
Moved to comments